index.html

```diff
@@ -9,20 +9,26 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <nav class="d-none mobile-nav" id="menu-nav">
+      <button id="close-btn" type="button" class="d-none menu-bars">
+        <i class="fas fa-times"></i>
+      </button>
+      <div class="menu-items">
+        <a class="menu-item" href="#projects">Portfolio</a>
+        <a class="menu-item" href="#about-me">About</a>
+        <a class="menu-item" href="#about-me">Contact</a>
+      </div>
+    </nav>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
```

Comment:
- [ ] Please use an anchor tag for the logo. You will notice that for most websites, the logo acts like the home button.